cobra_test.go

```diff
@@ -916,15 +916,14 @@ func TestRootUnknownCommand(t *testing.T) {
 
 func TestRootUnknownCommandSilenced(t *testing.T) {
 	r := noRRSetupTestSilenced("bogus")
-	s := "Run 'cobra-test --help' for usage.\n"
 
 	if r.Output != "" {
-		t.Errorf("Unexpected response.\nExpecting to be: \n\"\"\n Got:\n %q\n", s, r.Output)
```

Comment:
BTW it's already fixed: https://github.com/spf13/cobra/commit/ec452170d21fadd9e927f24ae471a258dc545261